docs/naps/3-spaces.md

```diff
@@ -0,0 +1,127 @@
+(nap3)=
+
+# NAP-3 — Spaces
+
+```{eval-rst}
+:Author: Lorenzo Gaifas, brisvag@gmail.com
+:Created: 2022-06-08
+:Resolution: <date resolved, in yyyy-mm-dd format>
+:Status: Draft
+:Type: Standards Track
+:Version effective: <version-number> (for accepted NAPs)
+``` 
+
+
+# Abstract
+
+`napari` is currently limited to holding (and rendering) data belonging to a single, universal space (which we often refer to as *world space*). However, it is often useful to have quick and easy access to different parts of a dataset that do no belong to the same coordinate system. In these cases, not only forcing data to live in the same world makes no sense, but it can make navigating the data and interacting with the viewer slower and less intuitive.
```

Comment:
```suggestion
`napari` is currently limited to holding (and rendering) data belonging to a single, universal space (which we often refer to as *world space*). However, it is often useful to have quick and easy access to different parts of a dataset that do not belong to the same coordinate system. In these cases, forcing data to live in the same world not only makes no sense, but it can make navigating the data and interacting with the viewer slower and less intuitive.
```

typo/grammar bot suggestion